release-notes.md

```diff
@@ -1,5 +1,16 @@
 # Release notes
 
+## July 31, 2019
+
+**Places Monitor 2.0.0**
+
+* Monitoring status is now persisted between launches.
+* Handling of the callback resulting from a location permission request no longer requires you to extend PlacesActivity.
```

Comment:
Please reword the 2nd point so that it reads:  "The handling of the callback, which resulted from a..."